app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: %i( index edit update destroy )
+  before_action :correct_user, only: %i( edit update )
+  before_action :load_user, only: %i( destroy update )
+
+  def index
+    @users = User.all.page(params[:page]).per(10)
```

Comment:
```suggestion
    @users = User.page(params[:page]).per Settings.abc_xyz
```
không cần all vẫn chạy được thì phải, với lại ném cái per kia vào Settings nhé :v